src/types/modules/react-native-svg.d.ts

```diff
@@ -0,0 +1,24 @@
+import {StyleProp} from 'react-native';
+import {CommonPathProps as BaseCommonPathProps, SvgProps as BaseSvgProps} from 'react-native-svg';
+
+declare module 'react-native-svg' {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+    interface SvgStyle extends ViewStyle {
```

Comment:
Should `SvgStyle` extend `ViewStyle`?

Reply:
It wasn't totally accurate, I modified it a bit